test/Cover/CoverGeneratorFactoryTest.php

```diff
@@ -31,15 +31,20 @@
 
 namespace OpusTest\Pdf\Cover;
 
-use Opus\Pdf\Cover\CoverGeneratorFactory;
-use Opus\Pdf\Cover\CoverGeneratorInterface;
+use Opus\Common\Config;
+use Opus\Common\Cover\CoverGeneratorFactory;
+use Opus\Common\Cover\CoverGeneratorInterface;
+use Opus\Pdf\Cover\DefaultCoverGenerator;
 use PHPUnit\Framework\TestCase;
 
 class CoverGeneratorFactoryTest extends TestCase
```

Comment:
Die Klasse ist jetzt in Common. Die Tests sollten dorthin verschoben werden. Es sollte dort Testfunktionen für eine nicht gesetzte Option und für eine unbekannte Klasse geben. Die Konfiguration dafür kann injiziert werden, mit `setConfig`. Da dort keine Implementation des Interfaces vorliegt, kann dieser Aspekt dort nicht getestet werden. Das ist okay (siehe `DefaultCoverGeneratorTest`).